<launch>
  <node pkg="lidar_detection" type="lidar_detection" name="lidar_detection" output="screen">
  </node>

  <arg name="open_rviz" default="true"/>
  <group if="$(arg open_rviz)"> 
    <node pkg="rviz" type="rviz" name="rviz" required="true"
          args="-d $(find lidar_detection)/rviz/sensor_view.rviz"/>
  </group>
  
</launch>
